pkg/interface/link/src/js/components/lib/invite-search.js

```diff
@@ -0,0 +1,369 @@
+              "list mono white-d f8 pv2 ph3 pointer" +
+              " hover-bg-gray4 hover-black-d"
+            }
+            onClick={e => this.addGroup(group)}>
+            {group}
+          </li>
+        );
+      });
+
+      let shipResults = state.searchResults.ships.map(ship => {
+        let nicknames = (this.state.contacts.has(ship))
+          ? this.state.contacts.get(ship).join(", ")
+          : "";
+        return (
+          <li
+            key={ship}
+            className={
+              "list mono white-d f8 pv1 ph3 pointer" +
+              " hover-bg-gray4 hover-black-d relative"
+            }
+            onClick={e => this.addShip(ship)}>
+            <Sigil
+              ship={"~" + ship}
+              size={24}
+              color="#000000"
+              classes="mix-blend-diff v-mid"
+            />
+            <span className="v-mid ml2 mw5 truncate dib">{"~" + ship}</span>
+            <span className="absolute right-1 di truncate mw4 inter f9 pt1">{nicknames}</span>
+          </li>
+        );
+      });
+
+      searchResults = (
+        <div
+          className={
+            "absolute bg-white bg-gray0-d white-d" +
+            " pv3 z-1 w-100 mt1 ba b--white-d overflow-y-scroll mh-16"
+          }>
+          {groupHeader}
+          {groupResults}
+          {shipHeader}
+          {shipResults}
+        </div>
+      );
+    }
+
+    let groupInvites = props.invites.groups || [];
+    let shipInvites = props.invites.ships || [];
+
+    if (groupInvites.length > 0 || shipInvites.length > 0) {
+      let groups = groupInvites.map(group => {
+        return (
+          <span
+            key={group}
+            className={
+              "f9 mono black pa2 bg-gray5 bg-gray1-d" +
+              " ba b--gray4 b--gray2-d white-d dib mr2 mt2 c-default"
+            }>
+            {group}
+            <span
+              className="white-d ml3 mono pointer"
+              onClick={e => this.deleteGroup(group)}>
+              x
+            </span>
+          </span>
+        );
+      });
+
+      let ships = shipInvites.map(ship => {
+        return (
+          <span
+            key={ship}
+            className={
+              "f9 mono black pa2 bg-gray5 bg-gray1-d" +
+              " ba b--gray4 b--gray2-d white-d dib mr2 mt2 c-default"
+            }>
+            {"~" + ship}
+            <span
+              className="white-d ml3 mono pointer"
+              onClick={e => this.deleteShip(ship)}>
+              x
+            </span>
+          </span>
+        );
+      });
+
+      participants = (
+        <div
+          className={
+            "f9 gray2 bb bl br b--gray3 b--gray2-d bg-gray0-d " +
+            "white-d pa3 db w-100 inter"
+          }>
+          <span className="db gray2">Participants</span>
+          {groups} {ships}
+        </div>
+      );
+    }
+
+    return (
+      <div className="relative">
+        <img
+          src="/~chat/img/search.png"
```

Comment:
You're going to want to copy in a search icon into the `pkg/arvo/link/img` folder, no cross-pollination of assets...

Also maybe copy a new version of this file in, some patches to it got put into os1-rc with #2370 I think.

Reply:
Ah, yeah. I copied this over from chat, should've checked for such things more thoroughly.